pin-to-pinata failing on production

```
> core-components@1.0.0 pin-to-pinata
> ts-node -P tsconfig.base.json ./scripts/pin-to-pinata.ts -e production -c production-137-0 -fd 1695632761000 -p *** -i ***/***

Error: Default tokens have not been set for this chainId=1
    at /home/runner/work/core-components/core-components/packages/common/src/configs.ts:227:13
    at Array.forEach (<anonymous>)
    at getEnvConfigs (/home/runner/work/core-components/core-components/packages/common/src/configs.ts:220:11)
    at getEnvConfigById (/home/runner/work/core-components/core-components/packages/common/src/configs.ts:251:19)
    at main (/home/runner/work/core-components/core-components/scripts/pin-to-pinata.ts:213:41)
    at Object.<anonymous> (/home/runner/work/core-components/core-components/scripts/pin-to-pinata.ts:4[67](https://github.com/bosonprotocol/core-components/actions/runs/6310415239/job/17132345903#step:9:68):1)
    at Module._compile (node:internal/modules/cjs/loader:1198:14)
    at Module.m._compile (/home/runner/work/core-components/core-components/node_modules/ts-node/src/index.ts:1618:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1252:10)
    at Object.require.extensions.<computed> [as .ts] (/home/runner/work/core-components/core-components/node_modules/ts-node/src/index.ts:1621:12)
Error: Process completed with exit code 1.
```

--- a/packages/common/src/mappings.ts
+++ b/packages/common/src/mappings.ts
@@ -197,5 +197,40 @@ export const chainIdToDefaultTokens = new Map<ChainId, Token[]>([
         decimals: "6"
       }
     ]
+  ],
+  [
+    1,
+    [
+      {
+        symbol: "ETH",
+        name: "ETH",
+        address: "0x0000000000000000000000000000000000000000",
+        decimals: "18"
+      },
+      {
+        symbol: "BOSON",
+        name: "Boson Token",
+        address: "0xC477D038d5420C6A9e0b031712f61c5120090de9",
+        decimals: "18"
+      },
+      {
+        symbol: "USDC",
+        name: "USDC",
+        address: "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+        decimals: "6"
+      },
+      {
+        symbol: "DAI",
+        name: "Dai Stablecoin",
+        address: "0x6B175474E89094C44Da98b954EedeAC495271d0F",
+        decimals: "18"
+      },
+      {
+        symbol: "USDT",
+        name: "Tether USD",
+        address: "0xdAC17F958D2ee523a2206206994597C13D831ec7",
+        decimals: "6"
+      }
+    ]
   ]
 ]);

--- a/packages/core-sdk/src/offers/queries.graphql
+++ b/packages/core-sdk/src/offers/queries.graphql
@@ -36,6 +36,58 @@ query getOffersQuery(
   }
 }
 
+query getOffersMediaQuery(
+  $offersSkip: Int
+  $offersFirst: Int
+  $offersOrderBy: Offer_orderBy
+  $offersOrderDirection: OrderDirection
+  $offersFilter: Offer_filter
+  $exchangesSkip: Int
+  $exchangesFirst: Int
+  $exchangesOrderBy: Exchange_orderBy
+  $exchangesOrderDirection: OrderDirection
+  $exchangesFilter: Exchange_filter
+  $includeExchanges: Boolean = false
+) {
+  offers(
+    skip: $offersSkip
+    first: $offersFirst
+    orderBy: $offersOrderBy
+    orderDirection: $offersOrderDirection
+    where: $offersFilter
+  ) {
+    id
+    metadata {
+      type
+      animationUrl
+      image
+      ... on ProductV1MetadataEntity {
+        productOverrides {
+          visuals_images {
+            url
+          }
+          visuals_videos {
+            url
+          }
+        }
+        product {
+          visuals_images {
+            url
+          }
+          visuals_videos {
+            url
+          }
+        }
+        productV1Seller {
+          images {
+            url
+          }
+        }
+      }
+    }
+  }
+}
+
 fragment OfferFields on Offer {
   ...BaseOfferFields
   exchanges(

--- a/packages/core-sdk/src/subgraph.ts
+++ b/packages/core-sdk/src/subgraph.ts
@@ -33333,6 +33333,71 @@ export type GetOffersQueryQuery = {
   }>;
 };
 
+export type GetOffersMediaQueryQueryVariables = Exact<{
+  offersSkip?: InputMaybe<Scalars["Int"]>;
+  offersFirst?: InputMaybe<Scalars["Int"]>;
+  offersOrderBy?: InputMaybe<Offer_OrderBy>;
+  offersOrderDirection?: InputMaybe<OrderDirection>;
+  offersFilter?: InputMaybe<Offer_Filter>;
+  exchangesSkip?: InputMaybe<Scalars["Int"]>;
+  exchangesFirst?: InputMaybe<Scalars["Int"]>;
+  exchangesOrderBy?: InputMaybe<Exchange_OrderBy>;
+  exchangesOrderDirection?: InputMaybe<OrderDirection>;
+  exchangesFilter?: InputMaybe<Exchange_Filter>;
+  includeExchanges?: InputMaybe<Scalars["Boolean"]>;
+}>;
+
+export type GetOffersMediaQueryQuery = {
+  __typename?: "Query";
+  offers: Array<{
+    __typename?: "Offer";
+    id: string;
+    metadata?:
+      | {
+          __typename?: "BaseMetadataEntity";
+          type: MetadataType;
+          animationUrl?: string | null;
+          image: string;
+        }
+      | {
+          __typename?: "ProductV1MetadataEntity";
+          type: MetadataType;
+          animationUrl?: string | null;
+          image: string;
+          productOverrides?: {
+            __typename?: "ProductV1ProductOverrides";
+            visuals_images: Array<{
+              __typename?: "ProductV1Media";
+              url: string;
+            }>;
+            visuals_videos?: Array<{
+              __typename?: "ProductV1Media";
+              url: string;
+            }> | null;
+          } | null;
+          product: {
+            __typename?: "ProductV1Product";
+            visuals_images: Array<{
+              __typename?: "ProductV1Media";
+              url: string;
+            }>;
+            visuals_videos?: Array<{
+              __typename?: "ProductV1Media";
+              url: string;
+            }> | null;
+          };
+          productV1Seller: {
+            __typename?: "ProductV1Seller";
+            images?: Array<{
+              __typename?: "ProductV1Media";
+              url: string;
+            }> | null;
+          };
+        }
+      | null;
+  }>;
+};
+
 export type OfferFieldsFragment = {
   __typename?: "Offer";
   id: string;
@@ -36163,6 +36228,59 @@ export const GetOffersQueryDocument = gql`
   }
   ${OfferFieldsFragmentDoc}
 `;
+export const GetOffersMediaQueryDocument = gql`
+  query getOffersMediaQuery(
+    $offersSkip: Int
+    $offersFirst: Int
+    $offersOrderBy: Offer_orderBy
+    $offersOrderDirection: OrderDirection
+    $offersFilter: Offer_filter
+    $exchangesSkip: Int
+    $exchangesFirst: Int
+    $exchangesOrderBy: Exchange_orderBy
+    $exchangesOrderDirection: OrderDirection
+    $exchangesFilter: Exchange_filter
+    $includeExchanges: Boolean = false
+  ) {
+    offers(
+      skip: $offersSkip
+      first: $offersFirst
+      orderBy: $offersOrderBy
+      orderDirection: $offersOrderDirection
+      where: $offersFilter
+    ) {
+      id
+      metadata {
+        type
+        animationUrl
+        image
+        ... on ProductV1MetadataEntity {
+          productOverrides {
+            visuals_images {
+              url
+            }
+            visuals_videos {
+              url
+            }
+          }
+          product {
+            visuals_images {
+              url
+            }
+            visuals_videos {
+              url
+            }
+          }
+          productV1Seller {
+            images {
+              url
+            }
+          }
+        }
+      }
+    }
+  }
+`;
 
 export type SdkFunctionWrapper = <T>(
   action: (requestHeaders?: Record<string, string>) => Promise<T>,
@@ -36566,6 +36684,21 @@ export function getSdk(
             { ...requestHeaders, ...wrappedRequestHeaders }
           ),
         "getOffersQuery",
+        "query"
+      );
+    },
+    getOffersMediaQuery(
+      variables?: GetOffersMediaQueryQueryVariables,
+      requestHeaders?: Dom.RequestInit["headers"]
+    ): Promise<GetOffersMediaQueryQuery> {
+      return withWrapper(
+        (wrappedRequestHeaders) =>
+          client.request<GetOffersMediaQueryQuery>(
+            GetOffersMediaQueryDocument,
+            variables,
+            { ...requestHeaders, ...wrappedRequestHeaders }
+          ),
+        "getOffersMediaQuery",
         "query"
       );
     }

--- a/scripts/pin-to-pinata.ts
+++ b/scripts/pin-to-pinata.ts
@@ -1,4 +1,3 @@
-import { GraphQLClient, gql } from "graphql-request";
 import { program } from "commander";
 import { CID } from "multiformats/cid";
 import axios, { AxiosRequestConfig } from "axios";
@@ -7,112 +6,10 @@ import { Readable } from "stream";
 
 import { BaseIpfsStorage } from "../packages/ipfs-storage/src/ipfs/base";
 import { EnvironmentType } from "../packages/common/src/types/configs";
-import { getEnvConfigById, offers, subgraph } from "../packages/core-sdk/src";
+import { getEnvConfigById, subgraph } from "../packages/core-sdk/src";
 import { buildInfuraHeaders } from "./utils/infura";
-import {
-  BaseAnimationMetadataFieldsFragmentDoc,
-  BaseConditionFieldsFragmentDoc,
-  BaseDisputeResolutionTermsEntityFieldsFragmentDoc,
-  BaseDisputeResolverFieldsFragmentDoc,
-  BaseExchangeTokenFieldsFragmentDoc,
-  BaseProductV1ExchangePolicyFieldsFragmentDoc,
-  BaseProductV1ProductFieldsFragmentDoc,
-  BaseProductV1SellerFieldsFragmentDoc,
-  BaseProductV1ShippingOptionFieldsFragmentDoc,
-  BaseProductV1VariationFieldsFragmentDoc,
-  BaseRangeFieldsFragmentDoc,
-  BaseSellerFieldsFragmentDoc,
-  GetOffersQueryQuery,
-  defaultWrapper
-} from "../packages/core-sdk/src/subgraph";
-const BaseOfferFieldsFragmentDoc = gql`
-  fragment BaseOfferFields on Offer {
-    id
-    metadata {
-      name
-      description
-      externalUrl
-      animationUrl
-      animationMetadata {
-        ...BaseAnimationMetadataFields
-      }
-      licenseUrl
-      condition
-      schemaUrl
-      type
-      image
-      ... on ProductV1MetadataEntity {
-        attributes {
-          traitType
-          value
-          displayType
-        }
-        createdAt
-        voided
-        validFromDate
-        validUntilDate
-        quantityAvailable
-        uuid
-        product {
-          ...BaseProductV1ProductFields
-        }
-        variations {
-          ...BaseProductV1VariationFields
-        }
-        productV1Seller {
-          ...BaseProductV1SellerFields
-        }
-        exchangePolicy {
-          ...BaseProductV1ExchangePolicyFields
-        }
-        shipping {
-          ...BaseProductV1ShippingOptionFields
-        }
-      }
-    }
-    range {
-      ...BaseRangeFields
-    }
-  }
-  ${BaseConditionFieldsFragmentDoc}
-  ${BaseSellerFieldsFragmentDoc}
-  ${BaseExchangeTokenFieldsFragmentDoc}
-  ${BaseDisputeResolverFieldsFragmentDoc}
-  ${BaseDisputeResolutionTermsEntityFieldsFragmentDoc}
-  ${BaseAnimationMetadataFieldsFragmentDoc}
-  ${BaseProductV1ProductFieldsFragmentDoc}
-  ${BaseProductV1VariationFieldsFragmentDoc}
-  ${BaseProductV1SellerFieldsFragmentDoc}
-  ${BaseProductV1ExchangePolicyFieldsFragmentDoc}
-  ${BaseProductV1ShippingOptionFieldsFragmentDoc}
-  ${BaseRangeFieldsFragmentDoc}
-`;
-const GetOffersQueryDocument = gql`
-  query getOffersQuery(
-    $offersSkip: Int
-    $offersFirst: Int
-    $offersOrderBy: Offer_orderBy
-    $offersOrderDirection: OrderDirection
-    $offersFilter: Offer_filter
-    $exchangesSkip: Int
-    $exchangesFirst: Int
-    $exchangesOrderBy: Exchange_orderBy
-    $exchangesOrderDirection: OrderDirection
-    $exchangesFilter: Exchange_filter
-    $includeExchanges: Boolean = false
-  ) {
-    offers(
-      skip: $offersSkip
-      first: $offersFirst
-      orderBy: $offersOrderBy
-      orderDirection: $offersOrderDirection
-      where: $offersFilter
-    ) {
-      ...BaseOfferFields
-    }
-  }
-  ${BaseOfferFieldsFragmentDoc}
-`;
+
+import { getSubgraphSdk } from "../packages/core-sdk/src/utils/graphql";
 
 const imageIpfsGatewayMap = {
   local: "https://test-permanent-fly-490.mypinata.cloud/ipfs/",
@@ -235,12 +132,14 @@ async function main() {
   const first = 100;
   let page = 0;
   let doMoreOffersExist = true;
-  let offersToProcess: subgraph.OfferFieldsFragment[] = [];
 
   console.log("\n1. Fetching offers to process...");
   const subgraphUrl = defaultConfig.subgraphUrl;
-  const client = new GraphQLClient(subgraphUrl);
-  const withWrapper = defaultWrapper;
+  const subgraphSdk = getSubgraphSdk(subgraphUrl);
+  let offersToProcess: Awaited<
+    ReturnType<typeof subgraphSdk.getOffersMediaQuery>
+  >["offers"] = [];
+  // const withWrapper = defaultWrapper;
   while (doMoreOffersExist) {
     const queryVars = {
       offersFirst: first,
@@ -254,32 +153,10 @@ async function main() {
         createdAt_lte: toTimestampSec
       }
     };
-    const paginatedOffers = (
-      await withWrapper(
-        (wrappedRequestHeaders) =>
-          client.request<GetOffersQueryQuery>(
-            GetOffersQueryDocument,
-            queryVars,
-            {
-              ...wrappedRequestHeaders
-            }
-          ),
-        "getOffersQuery",
-        "query"
-      )
-    ).offers;
+    const { offers: paginatedOffers } = await subgraphSdk.getOffersMediaQuery(
+      queryVars
+    );
 
-    // const paginatedOffers = await offers.subgraph.getOffers(subgraphUrl, {
-    //   offersFirst: first,
-    //   offersSkip: page * first,
-    //   offersOrderDirection: subgraph.OrderDirection.Asc,
-    //   offersOrderBy: subgraph.Offer_OrderBy.CreatedAt,
-    //   offersFilter: {
-    //     disputeResolverId: envName === "testing" ? "3" : undefined,
-    //     id_in: list ? list.split(",") : undefined,
-    //     createdAt_gte: fromTimestampSec
-    //   }
-    // });
     offersToProcess = [...offersToProcess, ...paginatedOffers];
 
     if (paginatedOffers.length < first) {

